### PR TITLE
Audiobook support

### PR DESCRIPTION
**Pull request by kristojorg**
Adds audiobook support to the `typeMap` and `MediaType`.

### CODE REVIEW

File: packages/opds-web-client/src/utils/file.ts [hunk 1]
```diff
@@ -44,6 +44,10 @@ export const typeMap: TypeMap = {
   "text/html;profile=http://librarysimplified.org/terms/profiles/streaming-media": {
     extension: "",
     name: "streaming-media"
+  },
+  "application/audiobook+json": {
+    extension: ".audiobook",
```
Comment (EdwinGuzman) [inline packages/opds-web-client/src/utils/file.ts:49]: @leonardr is this right?

Comment (leonardr) [reply packages/opds-web-client/src/utils/file.ts:49]: I think `.audiobook-manifest` is more accurate. That's what we use in core/model/constants.py and that's what the example has in https://github.com/readium/webpub-manifest/blob/master/extensions/audiobook.md.

**Comment by kristojorg** [comment]
I believe we've decided to leave this as `.audiobook`. In which case, is there anything blocking this merge?

**Comment by kristojorg** [comment]
Do you mind merging? I'm not authorized here

**Pull request merged**